### ISSUE TITLE
Parse Release Term "HBOMaxHD" as WEBDL, rather than HDTV 

### ISSUE THREAD

**Issue by PartTheSeas** [Sonarr/Sonarr#5489]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Not a big problem, just the parsing of release titles is wrong. "HBOMaxHD" is very popular, specifically for German Releases. As of now, it gets parsed as HDTV, but instead it should be parsed as "WEBDL"

### Describe the solution you'd like

Parse "HBOMaxHD" as WEBDL, just like "AmazonHD" for instance

### Describe alternatives you've considered

Manually retagging/renaming, which is pretty inconvenient if you have a lot of those files

**Comment by PearsonFlyer** [comment]
Please provide example, full, release names.

**Comment by PartTheSeas** [comment]
> 

Of course! For instance: The.Time.Travelers.Wife.S01E01.Erste.Begegnungen.German.DD51.Synced.DL.1080p.HBOMaxHD.AVC-TVS
